docs/configure.md

```diff
@@ -86,21 +86,21 @@ label: my_import
 ```yaml
 email_report:
     email_from: import@lephare.com
-    recipients: lephare_import.email_report.recipients
-    subject_pattern: "[%status%] Rapport d'import my_import"
+    recipients: <preset_name>
+    subject_pattern: "[%status%] Import report : %name%"
     email_template: null
 ```
 
-| Name               | Mandatory (default)                     | Type                   |                                     |
-| ----------------- |-----------------------------------------| ---------------------- |-------------------------------------|
-| `email_from`      | ✅                                       | ?string                | Sender's email address              |
-| `recipients`      | ➖ (`[]`)                                | string[] &#124; string | Email(s) of recipient(s)            |
-| `subject_pattern` | ➖ (`[%status%] Import report : %name%`) | string                 | Mail subject                        |
-| `email_template`  | ➖ (`null`)                              | ?string                | Path to a Twig email template file  |
+| Name              | Mandatory (default)                      | Type                   |                                                                                    |
+| ----------------- | ---------------------------------------- | ---------------------- | ---------------------------------------------------------------------------------- |
+| `email_from`      | ✅                                       | ?string                | Sender's email address                                                             |
+| `recipients`      | ➖ (`[]`)                                | string[] &#124; string | Email(s) of recipient(s) or preset name defined in bundle configuration            |
```

Comment:
Suggestion : add a (relative) Markdown link to https://github.com/le-phare/import-bundle/blob/master/docs/configure.md#bundle-configuration